src/main/java/com/easypost/model/Tracker.java

```diff
@@ -157,6 +265,8 @@ public void setTrackingDetails(final List<TrackingDetail> trackingDetails) {
         this.trackingDetails = trackingDetails;
     }
 
+    // This method is a misspelling, but it persists to avoid breaking backwards compatibility
+
```

Comment:
This comment needs to be tied to the `getUpdateAt` method and not free-floating as it is now.

Reply:
Resolved in 28156240aa91125dfeeb163cd5eb6d71869b762a